sections/schema.js

```diff
@@ -302,8 +302,8 @@ export default [
   {
     type: "method",
     method: "datetime",
-    example: "table.datetime(name, [precision])",
-    description: "Adds a datetime column, with optional precision for MySQL.",
+    example: "table.datetime(name, [useTz], [precision])",
```

Comment:
Isnt this documentation still wrong, because now those parameters should be given as options object and old parameters are left undocumented just to not break migrations backwards compatibility?

Reply:
Yeah, I know, this still wasn't updated after last batch of changes, will fix when I have next timeslot :)